docs/resources/cce_node_v3.md

```diff
@@ -65,6 +65,11 @@ The following arguments are supported:
 
 * `annotations` - (Optional) Node annotation, key/value pair format. Changing this parameter will create a new resource.
 
+* `taints` - (Optional) Taints to created nodes to configure anti-affinity.
+  * `key` - (Required) A key must contain 1 to 63 characters starting with a letter or digit. Only letters, digits, hyphens (-), underscores (_), and periods (.) are allowed. A DNS subdomain name can be used as the prefix of a key.
+  * `value` - (Required) A value must start with a letter or digit and can contain a maximum of 63 characters, including letters, digits, hyphens (-), underscores (_), and periods (.).
```

Comment:
Please emphasize values with `backquotes`

Reply:
Can you elaborate on that please? What do you mean, when you are referring to values? The arguments (as Terraform calls them) like `taints`, `key` and `value` are already enclosed in backticks/backquotes.

The docs were copied from cce_node_pool_v3, so if something is wrong here it is also wrong there.

Reply:
I was meaning quoting also the following:
```
A key must contain `1` to `63` characters...
Only letters, digits, hyphens (`-`), underscores (`_`), and periods (`.`) are allowed.
```
I suppose we don't always do like that ourselves :) 

P.S.
Actually, it's done only in 4 resources against 33 without backquoting such values. For CCE it's not done in other resources docs. So it's okay as it's now.